patch/cisco-ds4424-null-of-node.patch

```diff
@@ -0,0 +1,23 @@
+commit 166b37e58be17655da39f3d7ad61014db4afa4a1
+Author: Billie R Alsup <balsup@cisco.com>
+Date:   Sun Aug 23 14:57:55 2020 -0700
+
+    Allow NULL of_node
+
```

Comment:
Please use `git format-patch`.

Reply:
As per the review comments patch updated to use git format-patch